### ISSUE TITLE
Move to Lerna

### ISSUE THREAD

**Issue by lucasbesen** [lucasbesen/react-kanban-dnd#6]
I'm planning a roadmap and Lerna will be very useful.

We can begin with: https://github.com/lucasbesen/react-kanban/issues/5

**Comment by lucasbesen** [comment]
Done on: https://github.com/lucasbesen/react-kanban-dnd/pull/26

**Comment by TRomesh** [comment]
@lucasbesen Will there be multiple packages in the future?. I hope you are referring to [lerna.js](https://lerna.js.org/) if so I'll start working on this.

**Comment by lucasbesen** [comment]
@TRomesh yes, in the future the idea is to have a new package with react-kanban-dnd + [Relay](https://github.com/facebook/relay). For now, we can start with two packages:

- react-kanban-dnd
- docz
